keyboards/preonic/keymaps/via/config.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
I definitely can, but this is basically a direct copy of the default folder so i am not sure who deserves credit - certainly not me! It has commits from both you and Jack that i noticed. All I did was remove the references to the startup songs for the removed keymaps (dvorak and colemak). Please advise on credit and I can add it!